src/src/rx_main.cpp

```diff
@@ -305,7 +305,11 @@ bool ICACHE_RAM_ATTR HandleFHSS()
 
     if (modresultTLM != 0 || ExpressLRS_currTlmDenom == 1) // if we are about to send a tlm response don't bother going back to rx
     {
+#if defined(Regulatory_Domain_EU_CE_2400)
+        BeginClearChannelAssessment();
```

Comment:
Will this clearChannelAssessment happen at the correct frequency every time? Are telemetry sometimes sent right after hopping to a new frequency?

Reply:
> Will this clearChannelAssessment happen at the correct frequency every time? Are telemetry sometimes sent right after hopping to a new frequency?

No they don't but I want to grab the time that we go into RX mode here so the wait does not have to be as long and may have actually passed, so no wait time may be needed, by the time we come to `ChannelIsClear`.

Reply:
As long as the wait-time with RX enabled happens at the same frequency as TX will happen, all is well. Will modresultTLM and  modresultFHSS never be both 0 at the same time? My understanding of this function is; if both are 0, the handleFHSS will hop to a new frequency with rx possibly enabled from before, but it will skip the beginCCA and go straight to TX on the new frequency without?

Reply:
As discussed on discord, the TLM is always sent as the first thing on a hop. Which is not ideal, but I can see why it's done. If you move it to slot 1 then it gets residual interference from the transmitter and always thinks the channel is in use and stops sending telemetry.